api-reference/data-collection-events/endpoints.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Endpoints"
+description: "DataCollectionEvent endpoints reference"
+---
+
+There is two types of data collection events: incoming and outgoing. 
+Edgee will store both incoming and outgoing data collection events for debugging and auditing purposes.
```

Comment:
@SachaMorard I would suggest adding something like "for up to X days" (to make sure it's clear we don't store all event data forever and it's intended just for debugging).